pkg/backup/item_backupper.go

```diff
@@ -160,6 +189,18 @@ func (ib *defaultItemBackupper) backupItem(logger logrus.FieldLogger, obj runtim
 	}
 	ib.backedUpItems[key] = struct{}{}
 
+	backup, msg, err := shouldBackup(obj, groupResource)
```

Comment:
i'd probably do away with the `msg` return value, and just log a generic "skipping completed <resourceType>" message based on the value of `backup`